pkg/accessibility/accessibility.go

```
@@ -0,0 +1,306 @@
+	if err != nil {
+		return true, err
+	}
+	req.Header.Set("Accept", "application/json")
+	if token != "" {
+		req.Header.Set("Authorization", "Bearer "+token)
+	}
+
+	client := &http.Client{Timeout: 5 * time.Second}
+	resp, err := client.Do(req)
+	if err != nil {
+		return true, fmt.Errorf("Bitbucket API request: %w", err)
```

golangci-lint (2.11.4)

[error] ST1005: error strings should not be capitalized

(staticcheck)